vmcloak/abstract.py

```diff
@@ -263,11 +265,13 @@ def buildiso(self, mount, newiso, bootstrap, tmp_dir=None):
             isocreate, "-quiet", "-b", "boot.img", "-o", newiso,
         ] + self.genisoargs + [outdir]
 
+        log.debug("Executing genisoimage: %s", " ".join(args))
         p = subprocess.Popen(
             args, stdout=subprocess.PIPE, stderr=subprocess.PIPE
         )
         out, err = p.communicate()
-        if p.wait() or out or err != GENISOIMAGE_ERROR:
+
+        if p.wait() or out or re.sub("[\s]+", " ", err).strip() not in GENISOIMAGE_WARNINGS:
```

Comment:
Same here, probably the easiest to put `warning = re.sub("[\s]+", " ", err).strip()` on the line above to make space.